packages/react-conformance/src/defaultTests.tsx

```diff
@@ -327,6 +327,129 @@ export const defaultTests: TestObject = {
     });
   },
 
+  'component-has-static-classnames-object': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
```

Comment:
None of these tests actually validate that the slot names are correctly in the classname right ?

I think we might need to actually use TS to parse the types and validate based on those. Something similar is done for the event handler validation

Reply:
There is a test that does this validation. Tagged you in a comment there.